src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -732,13 +732,14 @@ private synchronized void update()
                 return adaptiveTrackProjection;
             }
 
+            final String endpointID = trackBitrateAllocation.endpointID;
```

Comment:
Maybe comment this? Otherwise the rationale and the problem aren't obvious, at least to me.